charts/alcide-advisor-cronjob/v2.1.0/app-readme.md

```diff
@@ -0,0 +1,48 @@
+# Alcide Kubernetes Advisor
```

Comment:
It would be good to create a README.md and copy paste everything from this file in it so it shows up in the "Detailed Descriptions" section of the UI. This file is meant to be used for a brief description of the chart and important notes (Similar to this section here between "Alcide Kubernetes Advisor" and "Pipeline Integrations").